docs/_includes/macros/card/template.njk

```diff
@@ -1,9 +1,67 @@
-{% from "macros/heading/macro.njk" import heading %}
+{% from "govuk/macros/attributes.njk" import govukAttributes -%}
+{% from "macros/heading/macro.njk" import heading -%}
 
-<article class="app-card">
-  {{ heading(params.heading) }}
+<!--
+Arguments to be passed:
+- layout: {string} "side-by-side", "full", "image-content".
+- reverse: {boolean} Reverse the layout of image-content variant, so image shows either left or right of content.
+- imageLink: {string} Absolute path of image location (e.g., assets/images/...)
+- imageClass: {string} Additional CSS class to be applied if needed.
+- imageAlt: {string} Alt text for image. If empty, the image will be treated as decorative for screen readers.
+- imageLinkLarge: {string} Absolute path for larger image location for x2 display (e.g., assets/images/...).
+-->
 
-  <div class="govuk-body">
-    {{ caller() }}
-  </div>
-</article>
+{%- macro _cardImage(params) %}
+  {% set classNames = "app-card__image govuk-!-margin-bottom-4" -%}
+
+  {% if params.imageClass %}
+    {% set classNames = classNames + " " + params.imageClass %}
+  {% endif -%}
+
+  <img {{- govukAttributes({
+    loading: "lazy",
```

Comment:
Do we need a `height` and `width` on the image to prevent layout shift?  Without it the browser can't determine the aspect ratio to know how much space to reserve before the image is loaded.